app/views/dealers/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<a href="/dealers">Dealer Index</a>
+
+<h1><%= @dealer.name %></h1>
+<p><%= @dealer.fully_staffed %></p>
+<p><%= @dealer.monthly_gross %></p>
+<p><%= @dealer.city %></p>
+<p><%= @dealer.created_at %></p>
+<p><%= @dealer.updated_at %></p>
```

Comment:
Do you think we need these?

Reply:
@abreaux26 we probably don't need them displayed actually, good call!